src/header/js/header-inline-title.js

```diff
@@ -82,7 +82,10 @@ export default () => {
 
 	const $closeBtn = $('button', {
 		class: `${prefix}-action-button ${prefix}-dismiss-button`,
-		aria: {labelledby: `${prefix}-action-button`}
+		aria: {
+			labelledby: `${prefix}-action-button`,
```

Comment:
This is kind of weird to have a `labelledby` and a `label`. Also the label lacks the ability to be translated?

Reply:
> When user agents compute the accessible name property of elements that have both an `aria-labelledby` attribute and an `aria-label` attribute, the user agents give precedence to `aria-labelledby`. ([MDN Docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute))